x-pack/plugins/apm/public/components/shared/service_link.tsx

```diff
@@ -35,22 +52,58 @@ export function ServiceLink({
     ? '/mobile-services/{serviceName}/overview'
     : '/services/{serviceName}/overview';
 
-  return (
-    <StyledLink
-      data-test-subj={`serviceLink_${agentName}`}
-      href={link(serviceLink, {
-        path: { serviceName },
-        query,
-      })}
-    >
-      <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+  if (serviceName === serviceDroppedBucketName) {
+    return (
+      <EuiFlexGroup alignItems="center" gutterSize="xs">
         <EuiFlexItem grow={false}>
-          <AgentIcon agentName={agentName} />
+          {i18n.translate('xpack.apm.serviceLink.other.label', {
+            defaultMessage: '_other',
```

Comment:
Perhaps @bmorelli25 is the best to ask here.

Reply:
I agree on the part that the translation for this is not required. Regarding the text, we wanted to display it as `_other` in order to avoid Name collision. A service can possibly have a name as `Other` which may cause collision.